Cargo.toml

```diff
@@ -7,3 +7,4 @@ edition = "2021"
 [dependencies]
 anyhow = "1.0.71"
 clap = { version = "4.2.7", features = ["derive"] }
+proc-macro2 = "^1.0.59"
```

Comment:
I think it would be sufficient to just update the lockfile (`cargo update -p proc-macro2`) rather than adding a not-directly-used dep here.

Reply:
Will fix this later, thanks.